docs/fides/docs/installation/docker.md

```diff
@@ -1,9 +1,9 @@
 # Installation from Docker
 
-For the ease of deployment in production, the community releases a production-ready reference container image.
+For the ease of deployment in production, the fidesctl community releases production-ready Docker Images, which are container reference images for running fidesctl. 
 
-The fidesctl community releases Docker Images which are reference images for fidesctl. Every time a new version of fidesctl is released, the images are available in the [ethyca/fidesctl DockerHub](https://hub.docker.com/r/ethyca/fidesctl/tags). There are also mid-release versions (dirty versions) that get uploaded to DockerHub on every commit to the main branch.
+Each time a new version of fidesctl is released, the updated images are available in the [ethyca/fidesctl DockerHub](https://hub.docker.com/r/ethyca/fidesctl/tags). There are also mid-release versions (dirty versions) that get uploaded to DockerHub on every commit to the main branch.
 
-These reference images contain all of the extras and dependencies for running the Python application. However they do not contain the required Postgres database.
+These reference images contain all of the extras and dependencies for running the Python application, but do not contain the required Postgres database, covered in this guide's [next steps](../installation/database/).
```

Comment:
does this work? I just ran the docs server on my local and this link goes to: `http://0.0.0.0:8000/fides/installation/installation/database/`